tests/cli-test.js

```diff
@@ -180,6 +180,48 @@ describe('CLI', function () {
     });
   });
 
+  describe(`--ignore-dependencies`, function () {
+    it('check console log', async function () {
+      const child = await runSupportedCmd([
+        `${__dirname}/fixtures/supported-project`,
+        `--ignored-dependencies @stefanpenner/a`,
+        `-i rsvp`,
+      ]);
+
+      expect(child).to.exitGracefully();
+      expect(child.stderr).to.includes(`Ignored: 2`);
+      expect(child.stderr).to.includes('✓ SemVer Policy');
+      expect(child.stdout).to.includes('Congrats!');
+    });
+
+    it('check if verbose do not incude the entry', async function () {
+      const child = await runSupportedCmd([
+        `${__dirname}/fixtures/supported-project`,
+        `--ignored-dependencies @stefanpenner/a`,
+        `-i rsvp`,
+        '--verbose',
+      ]);
+
+      expect(child).to.exitGracefully();
+      expect(child.stderr).to.includes(`Ignored: 2`);
+      expect(child.stderr).to.includes('✓ SemVer Policy');
+      expect(child.stdout).to.includes('Congrats!');
+      expect(child.stdout).not.include(
+        '@stefanpenner/a                    1.0.3                          2.0.0   major',
```

Comment:
is it possible to use a more general assertion here, maybe a regex, so that format changes don't break things

Reply:
we can probably do a regex to match the blank spaces, but package name, versions, and type of violation must match.